ansible/roles_ocp_workloads/ocp4_workload_cyberark_dap/.yamllint

```diff
@@ -0,0 +1,14 @@
+---
+extends: default
+
+rules:
+  comments:
+    require-starting-space: false
+  comments-indentation: disable
+  indentation:
+    level: warning
+    indent-sequences: consistent
+  line-length:
+    level: warning
+    max: 325
+    allow-non-breakable-inline-mappings: true
```

Comment:
We already have a `.yamllint` at `ansible/roles_ocp_workloads/.yamllint` that covers this directory. I don't think any need to create another within the role.